go.mod

```diff
@@ -58,7 +58,7 @@ require (
 	github.com/xwb1989/sqlparser v0.0.0-20180606152119-120387863bf2
 	go.opencensus.io v0.22.5 // indirect
 	go.uber.org/atomic v1.5.1
-	go.uber.org/cadence v0.16.1-0.20210330222409-fcc703e94e2f
+	go.uber.org/cadence v0.16.1-0.20210409233821-65903f75c15d
```

Comment:
I have these lines even after `go mod tidy`. Should I do manually revert them?

Reply:
nit: maybe update to the latest client commit?